index.d.ts

```diff
@@ -444,6 +444,54 @@ declare module 'plaid' {
     accounts: Array<AccountWithOwners>;
   }
 
+  interface LiabilitiesResponse extends AccountsResponse {
+    liabilities: {
+      student: Array<{
```

Comment:
@mlai I think we should have a top level typing for this object, so that clients can refer to it in their own code (otherwise you have to redefine it).

Reply:
https://github.com/plaid/plaid-node/pull/212/commits/c633d6bff5954c65114d63587bfcabcb9a064a2c